app/assets/stylesheets/pages/home.scss

```diff
@@ -0,0 +1,91 @@
+/* HomePage Styles */
+
+.homePage {
+  align-items: center;
+
+  .title-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: $orange60;
+    background-image: url(asset-path('stencils/instrument_white.png'));
+    width: 100vw;
+    background-size: 85vw auto;
```

Comment:
I don't have your app locally so I can't check myself, but when using percentage based things like `vw` and `vh`, make sure you resize your browser frequently to make sure things don't get weird on smaller or larger screens